packages/back-end/src/models/ReportModel.ts

```diff
@@ -26,6 +27,14 @@ type ReportDocument = mongoose.Document & ReportInterface;
 
 const ReportModel = mongoose.model<ReportDocument>("Report", reportSchema);
 
+const toInterface = (doc: ReportDocument): ReportInterface => {
```

Comment:
nit: should we rename this to make it clear it's about migrating from legacy to current experiment interfaces? is there any reason to think about how this will be used again in the near future to handle other migrations?